SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -80,4 +84,4 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
     If you have `UActorComponent`s which don't extend `USceneComponent` but exist in multiple on an Actor, *this can potentially crash your game*. Make sure there are never multiple components of the same type on an Actor. If there are make sure you modify your Actor to only have one, or use the explicit `SpatialType` tagging approach and don't tag the duplicate `UActorComponent`.
 
     * **Output path for the generated schemas**</br>
-    Use this to specify the filepath of ICG-generated schema file.  If you leave this empty, it defaults to `<ProjectFileLocation>/../spatial/schema/improbable/unreal/generated/`.
+    Use this to specify the filepath of ICG-generated schema file.  If you leave this empty, it defaults to `<GameRoot>/../spatial/schema/improbable/unreal/generated/`.
```

Comment:
ICG is no longer a thing, let's just call it "path of the generated schema file"